content/blog/azure-native-v3/index.md

```diff
@@ -0,0 +1,602 @@
+---
+title: "Azure Native V3: A Leaner, Faster SDK for Microsoft Azure"
+date: 2025-04-03
+draft: false
+meta_desc: Azure Native V3 delivers 75% smaller SDK size, updated API versions, and flexible options for working with explicit API versions in Microsoft Azure.
+meta_image: meta.png
+authors:
+    - thomas-kappler
+    - meagan-cojocar
+
+tags:
+    - azure-native
+    - azure
+    - features
+    - releases
+
+social:
+    twitter:
+    linkedin: 
+---
+
+Today we're excited to announce the release of Pulumi Azure Native V3, a major update to our Azure Native provider that improves developer experience through significant SDK size reduction, up-to-date default API versions, and a flexible approach to managing explicit API versions.
```

Comment:
Can we mention the 75% figure sooner and more prominently? It is remarkably impressive.